doc/contriburting_to_eva/github_actions.md

```diff
@@ -0,0 +1,57 @@
+# Automated testing with GitHub Actions
+When contributing to eva, a number of automated tests are performed by GitHub Actions to ensure that tests pass successfully, and that code and configuration files follow style conventions.
+
+An overview of each GitHub Actions workflow is provided in the following subsections.
+
+## Python coding norms
```

Comment:
You might add that the coding norms tests can also be run at the command line in the top level directory (>python pycodestyle_run.py).

Reply:
good suggestion, thanks!